src/tangent_types/thunks.jl

```diff
@@ -249,3 +249,121 @@ function Base.show(io::IO, x::InplaceableThunk)
     show(io, x.val)
     print(io, ")")
 end
+
+
+"""
+    BroadcastThunk{T}(bc::Broadcasted)
+
+A new kind of thunk, which wraps Base's lazy broadcasting. `T` is the eltype.
```

Comment:
It is only new til it isn't.
We can talk about it being `new` in the release notes
```suggestion
A kind of thunk, which wraps Base's lazy broadcasting. `T` is the eltype.
```